src/renderer/atlas/FontCache.h

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include <dwrite_3.h>
+
+namespace Microsoft::Console::Render::Atlas::FontCache
```

Comment:
I really didn't know where to put this stuff, so I put it here in the hopes we'll remove DxEngine one day and then it's just ~me and you~ AtlasEngine and two calls from WT-land.

Reply:
Maybe we should put this into the DxEngine project so that we can backport it to 1.12?

Reply:
Yes please. I'd rather it be somewhere easy to backport. :)